packages/cms/src/components/sections/Hero.jsx

```diff
@@ -222,7 +222,9 @@ class Hero extends Component {
         <div className="cp-section-inner cp-hero-inner">
           {/* caption */}
           <div className="cp-section-content cp-hero-caption">
-            {heading}
+            <div className="cp-hero-heading-wrapper">
+              {heading}
+            </div>
```

Comment:
I think I'd rather have you change the `<Fragment>` code above, on lines 212-217, to be this `<div>`. What do you think?